spacy/tests/regression/test_issue4104.py

```diff
@@ -0,0 +1,13 @@
+# coding: utf8
+from __future__ import unicode_literals
+
+import pytest
+from ..util import get_doc
+
+@pytest.mark.parametrize('text', ['dry spun spun-dry'])
```

Comment:
Not sure we need to parametrize here because the expected values are hard-coded into the test anyways. So there's not really a motivation to try out different words here. So feel free to move the string `'dry spun spun-dry'` into the function.